bin/alluxio-start.sh

```diff
@@ -85,25 +83,26 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      is_ram_folder_mounted
+      local folder_type=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.alias)
+      local ram_folder=$(${BIN}/alluxio getConf alluxio.worker.tieredstore.level0.dirs.path)
+      if [[ ${folder_type} != "MEM" ]]; then
```

Comment:
Is there any guarantee that the alias has to be called `MEM`? If so, is there any documentation saying that the ram must be the top tier, and it must be called `MEM`?